<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDemsList</class>
 <widget class="QWidget" name="IDemsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineFilter">
     <property name="toolTip">
      <string>Filter: Start to type and the list will be reduced to matching items.</string>
     </property>
     <property name="placeholderText">
      <string>start typing...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CDemTreeWidget" name="treeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/icons/48x48/Help.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelHelpFillMapList">
         <property name="text">
          <string>To add files with elevation data use &lt;a href='DemFolders'&gt;File-&gt;Setup DEM Paths&lt;/a&gt;.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHelpActivateMap">
         <property name="text">
          <string>Use the context menu (right mouse button click on entry) to activate a file. Use drag-n-drop to move the activated file in the process order. </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionActivate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/Check.png</normaloff>
     <normalon>:/icons/32x32/Cancel.png</normalon>:/icons/32x32/Check.png</iconset>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/Up.png</normaloff>:/icons/32x32/Up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Hide DEM behind previous one</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/Down.png</normaloff>:/icons/32x32/Down.png</iconset>
   </property>
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="toolTip">
    <string>Show DEM on top of next one</string>
   </property>
  </action>
  <action name="actionReloadDem">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/Reset.png</normaloff>:/icons/32x32/Reset.png</iconset>
   </property>
   <property name="text">
    <string>Reload DEM</string>
   </property>
  </action>
  <action name="actionClearFilter">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/32x32/Filter.png</normaloff>:/icons/32x32/Filter.png</iconset>
   </property>
   <property name="text">
    <string>Clear Filter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDemTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>dem/CDemList.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClearFilter</sender>
   <signal>triggered()</signal>
   <receiver>lineFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
